docs/user-guide/docs-source/setup.md

```diff
@@ -0,0 +1,123 @@
+# Setup
+
+
+
+## Initial Setup (One-Time)
+
+Complete the following setup before using Ambit for the first time.
+
+>  ⏱ Initial setup takes about 20 minutes.
+
+> 🛑 These instructions assume you already have Unreal Engine 4.27 installed. If you don't, you can [download Unreal Engine for free](https://www.unrealengine.com/en-US/download). Choose the Unreal Engine license that is appropriate for your team. Ambit will work with any Unreal Engine license.
+
+### 1. Download the Ambit plugin for UE4
+
+Download the latest pre-built release of the Ambit UE4 plugin from the GitHub [Releases section](https://github.com/aws-samples/aws-ambit-scenario-designer-ue4/releases).
+
+Unzip the file you downloaded. You may unzip it to any location that's convenient for you. It will unzip as a folder named "AwsAmbit".
+
+We'll return to the Ambit plugin in later steps.
```

Comment:
Minor: It might be useful to mention that steps 2 and 3 are for setting up AWS related stuff, and if you already have these setup you can skip to step 4.

Reply:
I think I'm going to leave this as-is for now so as to avoid sprinkling too many conditional statements in our docs. I'm going to have faith that if someone already has the AWS CLI installed they'll recognize that this step (for example) doesn't apply to them.